### PR TITLE
Insert a row for each CVE found in Grype JSON

### PR DESCRIPTION
**Pull request by jdolitsky**
Note: do not merge until:
1. new column added to BigQuery table(s)
2. Secret for `GCLOUD_TABLE_VULNS` added to this repo
3. `go run cmd/tableinit/main.go` is run with table name above

### CODE REVIEW

File: cmd/tableinit/main.go
Comment (jspeed-meyers) [inline cmd/tableinit/main.go:14]: It might be worth putting a comment here explaining what `GcloudTable` is just to help our future selves. I could see myself being confused in six months when I read this code again.

File: main.go
Comment (jspeed-meyers) [inline main.go:108]: This block looks great, but @luhring is def the world's expert :)

**Review by jspeed-meyers — APPROVED** [review]
LGTM. Like usual, wow. Question: Do you plan to (in a separate PR) add this row by row data collection for `trivy` too or only `grype`? No pressure any which way. Just curious.

File: main.go
Comment (luhring) [inline main.go:110]: This isn't a huge deal, but I'd suggest pulling these from `.artifact.name` and `.artifact.version` instead. This is the "source of truth" in terms of the matched package. As a bonus, this change might also mean we don't need the inner loop on `for _, detail := range match.MatchDetails`.

Comment (luhring) [reply main.go:110]: Also there's a decent chance I'm not following this exactly right. I see we define another `types.Vuln` down below, too 🤔

Comment (jdolitsky) [reply main.go:110]: for some reason this does not result in the same number/combo of results displayed by the grype in a given scan. both appear to be needed to get the total

Comment (jdolitsky) [reply main.go]: I took another pass to simplify! please check again

**Comment by jdolitsky** [comment]
> Do you plan to (in a separate PR) add this row by row data collection for trivy too or only grype? No pressure any which way. Just curious.

Was not planning at this point to add anything for Trivy output, but we could do that later in a separate table (or try to combine)

File: cmd/tableinit/main.go
Comment (jdolitsky) [reply cmd/tableinit/main.go]: yep! added some comments

**Review by luhring — APPROVED** [review]
LGTM!

**Comment by jspeed-meyers** [comment]
Over to you @jdolitsky!

**Comment by jdolitsky** [comment]
Let's wait until next week to merge this

**Comment by jspeed-meyers** [comment]
@jdolitsky and I can merge on Monday and make sure we don't wreak too much havoc.

**Pull request merged**
